cypress/suites/dashboard.suite.ts

```diff
@@ -122,27 +177,383 @@ export default {
+
+        it("should produce no results for an invalid search by asset Id", () => {
+          cy.get("div").find(">label").parent().type(invalidDexWidgetSearchByAssetId);
+          cy.get("li").should("not.exist");
+        });
+
+        it("should provide bid/ask order info", () => {
+          cy.get("div").find(">label").parent().type(validDexWidgetSearchByAssetId);
+          cy.contains(validSearchByAssetIdResult).click();
+          cy.contains(highestBidOrderPrice).should("exist");
+          cy.contains(lowestAskOrderPrice).should("exist");
+        });
+
+        it("should not open collection dialog when inputs aren't filled out", () => {
+          cy.get("div").find(">label").parent().type(validDexWidgetSearchByAssetId);
+          cy.contains(validSearchByAssetIdResult).click();
+          cy.get("button").contains("Buy").click();
+          cy.get("button").contains("Sell").click();
+          cy.contains(isSeedPhraseCollectionOpen).should("not.exist");
+        });
+
+        it("on buy, should open collection dialog when inputs are properly filled out", () => {
+          cy.get("div").find(">label").parent().type(validDexWidgetSearchByAssetId);
+          cy.contains(validSearchByAssetIdResult).click();
+          cy.get('input[placeholder*="Price"]').type("1");
+          cy.get('input[placeholder*="Quantity"]').eq(0).type("1");
+
+          cy.get("button").contains("Buy").click();
+          cy.contains(isSeedPhraseCollectionOpen).should("exist");
+        });
+
+        it("on sell, should open collection dialog when inputs are properly filled out", () => {
+          cy.get("div").find(">label").parent().type(validDexWidgetSearchByAssetId);
+          cy.contains(validSearchByAssetIdResult).click();
+          cy.get('input[placeholder*="Price"]').type("1");
+          cy.get('input[placeholder*="Quantity"]').eq(0).type("1");
+
+          cy.get("button").contains("Sell").click();
+          cy.contains(isSeedPhraseCollectionOpen).should("exist");
+        });
+
+        // order message is not correct, currently it's messageText.transaction.success which must be a defect in the useAPIRouter()
+        it("should execute a buy properly", () => {
+          cy.get("div").find(">label").parent().type(validDexWidgetSearchByAssetId);
+          cy.contains(validSearchByAssetIdResult).click();
+          cy.get('input[placeholder*="Price"]').type("1");
+          cy.get('input[placeholder*="Quantity"]').eq(0).type("1");
+
+          cy.get("button").contains("Buy").click();
+          cy.get('input[placeholder*="Enter Seed Phrase"]').type(testnetSeedPhrase);
+          cy.get("button").contains("Confirm & Send").click();
+          cy.get("#notistack-snackbar").should("contain.text", messageText.orders.success); // snackbar
+        });
+
+        // order message is not correct, currently it's messageText.transaction.success which must be a defect in the useAPIRouter()
+        it("should execute a sell properly", () => {
```

Comment:
They're all testnet tests now (the login address is from testnet). However, many of the tests will work on either net and it would still be useful to be able to eliminate any tests which require spending testJUP. I'll make an issue. 

https://github.com/Up-North-Coding/jovian/issues/79